### DISCUSSION TITLE
Coastline Prediction using Existing Climate Change Models (Project 229)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#60] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #229](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/edit/main/projects/Coastline%20Prediction%20using%20Existing%20Climate%20Change%20Models)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by eliseogarza** [comment]
Hi, my name is Eliseo Garza and I'm a student currently studying at Texas A&M University. I'm interested in working and collaborating with others on this project over the course of the summer.

**Reply by robertogl** [reply]
Hello @eliseogarza,

Great to know you will be working on this project! Please do not hesitate to ask any question using this discussion.

**Reply by sramharack** [reply]
Good day, myself and some researchers from UWI St. Augustine would like to collaborate with you. Is this possible?

@eliseogarza

**Reply by robertogl** [reply]
Hello @shanks847,

It looks like @eliseogarza posted this message quite some time ago and might have dropped the project. We received a couple of submissions to this project that could be good starting points for you, especially the latest one with the app. Perhaps you and your team can work on improving such an app and extend its features. I can certainly provide help if you need.

Best,
Roberto

**Comment by bala35210** [comment]
Hi, my name is Balakumaran Selvaraji and I'm a student currently studying at CEG, Anna University. I have a team of 4 who will be working and collaborating with others on this project.

**Comment by wjhuBU** [comment]
Hello, we are a team of 4 undergraduate seniors from Boston University, currently studying Computer & Electrical Engineering. We are interested in working on this project over the course of the school year (Fall 2022 to Spring 2023) and would like to collaborate with others.

**Comment by sramharack** [comment]
I'll look into it, however I'm also interested in attempting some of the
other challenges and would greatly appreciate any direction you could
provide.

I would love to have a chat if possible.

Regards
S.R

**Reply by robertogl** [reply]
Sounds good, I can send a meeting invite. Do you have any day/time constraint?

**Comment by sramharack** [comment]
Anytime within the next two weeks (14-28th) after 6PM GMT-4 works for me.

**Reply by robertogl** [reply]
Is there any day within that 2-week period you may be available before 5pm?

**Comment by sramharack** [comment]
Thursday 31st at 4PM GMT-4
